src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -844,67 +864,6 @@ private void ParseHelper(string[] args)
                         break;
                     }
                 }
-#if DEBUG
-                // this option is useful when debugging ConsoleHost remotely using VS remote debugging, as you can only
-                // attach to an already running process with that debugger.
-                else if (MatchSwitch(switchKey, "wait", "wa"))
-                {
-                    // This does not need to be localized: its chk only
-
-                    ((ConsoleHostUserInterface)_hostUI).WriteToConsole("Waiting - type enter to continue:", false);
-                    _hostUI.ReadLine();
-                }
-
-                // this option is useful for testing the initial InitialSessionState experience
-                else if (MatchSwitch(switchKey, "iss", "iss"))
-                {
-                    // Just toss this option, it was processed earlier...
-                }
-
-                // this option is useful for testing the initial InitialSessionState experience
-                // this is independent of the normal wait switch because configuration processing
-                // happens so early in the cycle...
-                else if (MatchSwitch(switchKey, "isswait", "isswait"))
```

Comment:
This needs to be kept. Otherwise, when using `-isswait` in debug build, it will result in an argument parsing error.

Reply:
Done.